src/operations/create_collection.ts

```diff
@@ -41,6 +41,17 @@ export interface TimeSeriesCollectionOptions extends Document {
   granularity?: 'seconds' | 'minutes' | 'hours' | string;
 }
 
+/** @public
+ * Configuration options for clustered collections
+ * @see https://www.mongodb.com/docs/v5.3/core/clustered-collections/
+ */
+export interface ClusteredCollectionOptions extends Document {
+  name?: string;
```

Comment:
Name is optional and if left out the index will be named by the server automatically.